### ISSUE TITLE
Prototype Update - State Change and Closing Request

### ISSUE THREAD

**Issue by JHarrietha-AOT** [bcgov/foi-flow#489]
Title of ticket: Prototype Update - State Change and Modals

Description
Updating prototype based off MIRO - Request State Workflow
https://miro.com/app/board/o9J_l9AKkos=/

Dependencies
- [x] Walkthrough of existing process for closing a request

DOD
- [x] Update Left Nav Dropdown to include all states included in Miro
- [x] Add, "Fee Estimate" "Oh Hold" "Harms Assessment" "Deduplication" to left nav bar
- [x] Create flow for closing a request - including reason why request is closing
- [x] Extensions (What are the reasons?) Add options with then dropdown of why and ability to input amount of days with will automatically update LDD

**Comment by JHarrietha-AOT** [comment]
@arielleandrews @lmullane I created this new UX issue just as a place to keep track of notes for updating the design to reflect all state changes in Miro including Flow for Closing a Request

**Comment by JHarrietha-AOT** [comment]
@arielleandrews I've added that quick walkthrough of closing a request in here so we can track it. I figured it would be quick it's just always helpful to see what the existing process is before I go and design something. Wondering if from your experience you think selecting a reason for closing from a dropdown menu is enough? Or do users need to be able to type out why they are closing the request.
